index.html

```diff
@@ -238,6 +244,84 @@ <h2>About me</h2>
           </ul>
         </div>
       </section>
+      <!-- Contact section -->
+      <footer id="contact">
+        <div class="contact-section">
+          <h3 class="contact-section-heading">
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h3>
+          <form action="https://formspree.io/f/xbjbloqv" method="post">
+            <ul>
+              <li>
+                <input
+                  class="user-info"
+                  name="name"
+                  type="text"
+                  id="user_name"
+                  placeholder="Full name"
+                  maxlength="30"
+                  required
+                />
+              </li>
+              <li>
+                <input
+                  class="user-info"
+                  name="email"
+                  type="email"
+                  placeholder="Email adress"
+                  required
+                />
+              </li>
+              <li>
+                <textarea
+                  class="input-message"
+                  name="user_message"
+                  cols="30"
+                  rows="10"
+                  maxlength="500"
+                  placeholder="Enter text here..."
+                  required
+                ></textarea>
+              </li>
+              <li class="submit-btn-container">
+                <button class="project-btn" type="submit">Get in touch</button>
+              </li>
```

Comment:
- [ ] [OPTIONAL] kindly make sure that Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template.
![image](https://user-images.githubusercontent.com/89282221/186500930-2bf71f12-ce32-44e0-8b08-520757514b3b.png)